src/stylesheets/screens/question.scss

```diff
@@ -0,0 +1,82 @@
+.questions {
+  width: 100vw;
+  height: 100vh;
+
+  &__header {
+    @include header-container();
+    height: rem(60px);
+
+    padding: rem(16px 16px 0);
+  }
+
+  &__quit-survey {
+    order: 2;
+    margin-left: auto;
+
+    cursor: pointer;
+  }
+
+  &__container {
+    @include details-container();
+    position: relative;
+
+    flex: 1 1 auto;
+    height: calc(100vh - #{rem(135px)});
+  }
+
+  &__details {
+    margin: auto 0;
+  }
+
+  &__number {
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'lg');
+    line-height: rem(20px);
+
+    opacity: 0.5;
+  }
+
+  &__title {
+    margin: 0;
+    padding-top: rem(10px);
+
+    color: map-get($brand-white, 'primary');
+    font-size: map_get($font-sizes, 'xxxl');
+    line-height: rem(41px);
+  }
+
+  &__footer {
+    display: flex;
+    flex: 0 1 auto;
+    width: 100%;
+    height: rem(75px);
+    padding-right: rem(16px);
```

Comment:
@rafayet-monon Can we change this to `rem(32px)`?
![Screen Shot 2563-11-24 at 18 22 31](https://user-images.githubusercontent.com/1772999/100088136-46992580-2e82-11eb-97d0-d184338d996b.png)